api/controllers/MerchStoreController.ts

```diff
@@ -14,6 +14,7 @@ import {
 import PermissionsService from '../../services/PermissionsService';
 import { UserAuthentication } from '../middleware/UserAuthentication';
 import {
+  ApiResponse,
   GetOneMerchCollectionResponse,
```

Comment:
nit: I don't think we should be importing `ApiResponse`, since the class was meant to be a general response type that other responses extend, and shouldn't be exposed to controller classes. You can create a new response type that just extends `ApiResponse` without any body, i.e.
```ts
export interface VerifyMerchOrderResponse extends ApiResponse {}
```